packages/rebound-runtime/lib/context.js

```diff
@@ -0,0 +1,104 @@
+import $ from "rebound-runtime/utils";
```

Comment:
Yanked this out of component's code for legibility. This will eventually be re-written as a single listenTo('all') callback and added to LazyValue, to be added to contexts where .addObserver is called.